cmd/login.go

```diff
@@ -43,6 +44,8 @@ func init() {
 			return nil
 		},
 		RunE: func(cmd *cobra.Command, args []string) error {
+			loginCmdArgs.certPass = os.Getenv("AZCOPY_SPA_CERT_PASSWORD")
```

Comment:
Minor: please follow the pattern and use the lifecycle manager to get the environment variables.